tests/exception/t16366.nim

```diff
@@ -0,0 +1,10 @@
+discard """
+  action: run
+  exitcode: 1
```

Comment:
why? `doAssertRaises(IOError):` should succeed, so exitcode should be 0; and since that's the default it should be:
```nim
discard """
  targets: "c cpp"
"""
```

Reply:
Fixed now, but it was because I don't know how testament works (I'm just copying and pasting) :)